akka-persistence-typed/src/main/scala/akka/persistence/typed/crdt/ORSet.scala

```diff
@@ -55,7 +62,7 @@ object ORSet {
             concatElementsMap(u.elementsMap.asInstanceOf[Map[A, Dot]]),
             underlying.vvector.merge(u.vvector)))
       case _: AtomicDeltaOp[A @unchecked] => DeltaGroup(Vector(this, that))
-      case DeltaGroup(ops)                => DeltaGroup(this +: ops)
+      case g: DeltaGroup[A @unchecked]    => DeltaGroup(this +: g.ops)
```

Comment:
@johanandren I removed the `case`

For some reason the match exhaustive checks don't work with these custom `unapply`. Maybe something with the type parameter? I rewrote them like this.